src/main/java/com/gtnewhorizons/angelica/compat/ModStatus.java

```diff
@@ -0,0 +1,25 @@
+package com.gtnewhorizons.angelica.compat;
+
+import cpw.mods.fml.common.Loader;
+
+public class ModStatus {
+    /**
+     * Mixin Version
+     */
+    public static boolean isNEIDLoaded;
+    /**
+     * ASM Version
+     */
```

Comment:
this comment looks like it applies to all following mods. Suggest `//ASM Version` at the end of the isOldNEIloaded line

Reply:
btw, that is a javadoc comment that appears when you mouse over the variable